sous-chef/pyproject.toml

```diff
@@ -1,31 +1,30 @@
 [tool.poetry]
 name = "sous_chef"
-version = "1.2.2"
+version = "1.2.3"
 description = "Plans your meals and grocery list"
 authors = ["Aurorian Clouds"]
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.11"
 fuzzywuzzy = "^0.18.0"
-google-api-python-client = "^2.63.0"
-google-auth-httplib2 = "^0.1.0"
-google-auth-oauthlib = "^0.5"
 hydra-core = "^1.2"
-# future version of pandas removes append
-pandas = "^1.3.5"
-pandera = "^0.11.0"
+pandas = "^1.3"
+pandera = "^0.11"
 pint = "^0.19"
-pygsheets = "^2.0.5"
+# issue with test__get_workbook_with_folder with 2.0.6
```

Comment:
@AkagiShigeru do you want to take a quick look or rather create a bug ticket to look at it another time?

Reply:
will create bug ticket